numpy/core/src/common/npy_partition.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __NPY_PARTITION_H__
+#define __NPY_PARTITION_H__
```

Comment:
We are slowly moving the guard to include the path, so
`NUMPY_CORE_SRC_COMMON_PARTITION_H_`.